skins/oasis/modules/ContributeMenuController.class.php

```diff
@@ -17,6 +17,8 @@ class ContributeMenuController extends WikiaController {
 	];
 
 	public function executeIndex() {
+		global $wgEnableFlowTracking;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgEnableFlowTracking" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)